mtggoldfish.py

```diff
@@ -148,16 +168,21 @@ def save_deck_to_cache(deck, deck_id):
     todays_date = datetime.now()
     month = todays_date.month
     day = todays_date.day
-    if month <= 9: month = "0%s" %(todays_date.month)
-    if day <= 9: day = "0%s" %(todays_date.day)
-    cache_file_name = "%s_%s_%s_%s" %(deck_id, month, day, todays_date.year)
+    if month <= 9:
+        month = "0%s" % (todays_date.month)
```

Comment:
I know this is the standard, but for conditionals that just have a single line of logic I thought to condense them in larger blocks of code like this to improve overall readability. It was a conscious decision I made. I won't say no to this pull request on that point though

Reply:
Single line multi-statements are generally discouraged. The reason is as you mean readability, but also, what happens if you suddenly want to include more logic after the if statement?